cmd/chaos-builder/main.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 var (
-	log = zap.Logger(true)
+	log = zap.New(zap.UseDevMode(true))
```

Comment:
why need to do this change?

Reply:
```golang
// Deprecated: use New() and the functional opts pattern instead:
//
// New(UseDevMode(development))
func Logger(development bool) logr.Logger {
	return LoggerTo(os.Stderr, development)
}
```

Reply:
is it suggested to set `development` = true?